src/main/java/com/checkout/payments/apm/BalotoSource.java

```diff
@@ -23,18 +22,9 @@
 
     private final String description;
 
-    @Data
-    @Builder
-    @NoArgsConstructor
-    @AllArgsConstructor
-    public static class Payer {
-        private String name;
-        private String email;
-    }
-
```

Comment:
I think `Payer` should still be here. I like the idea or reusing it, but `BalotoSource` payer does not have the attribute `document` for example. In practice it should be ok, but I would prefer to have clear/specific payer objects.